docs/Containers/Container-Tasks.md

```diff
@@ -0,0 +1,279 @@
+leveraging the advantages of native containerization, enabling better
+speed, flexibility, and efficiency for running tasks.
+
+## What's Different
+
+While tasks running on containers come with the same general
+capabilities you're familiar with in regular host tasks, ultimately they
+are not hosts themselves, meaning certain low-level system processes
+such as systemd present on the host that may not be readily available
+within a container. To learn more, the [docker security
+documentation](https://docs.docker.com/engine/security/) explains the
+features of Docker's isolation model. There are a couple of differences
+to note as we roll out this new feature:
+
+1.  Task Groups: The initial release of container tasks does not support
+configuring task groups. This is something we aim to support in
+future iterations.
+
+2.  Priority: The priority setting feature will not be available during
+the initial release for container tasks.
+
+3.  Greater configurability: When running container tasks, resources
+such as CPU and memory usage must be explicitly configured, unlike
+in the current distro model. Users are also responsible for
+picking the image to use, and all required software is downloaded
+during runtime. Furthermore, the container your task runs on is
+dedicated solely to that task and is never reused, so you are free
+to do whatever you want with it without needing to worry about
+leaving the environment in a messy state for the next task.
+
+# How to get started
+## Image lifecycle and self service
+
+At the moment, our image security automation is still [under
+development](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit#heading=h.ghiwibz2opmd),
+so for the time being, modifying a project\'s respective Dockerfile
+requires the submission of a PR against our repo.
+
+1.  Stage a PR in our centralized image
+    [repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
+    This repo hosts Dockerfiles for all teams participating in the
+    beta of running container tasks in Evergreen. The PR, when merged
+    into the main branch, triggers an automated Drone pipeline which
+    then builds the image and pushes it to Amazon Elastic Container
+    Registry (ECR). To onboard, your PR changes must include a
+    relevant Dockerfile, created under a new directory that has the
+    name of your project. Review should be requested from
+    **evergreen-ci/evg-app**, and the PR's approval will be contingent
+    on the Dockerfile following the [secure image
+    policy](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit)
+    that we currently have in place.
+
+2.  Once the PR has been approved and is merged into main with the Drone
+    build passing, you may reference the URI of the newly built image.
+    Its format will be as follows:
+    
+    > **557821124784.dkr.ecr.us-east-1.amazonaws.com/evergreen/${directory}:${sha}**
```

Comment:
Since this is the ECR registry that our prod cluster is using, all teams that run container tasks will have to put this url with out AWS account, so I presume it'd need to exist somewhere in our docs?

One solution if we want to avoid this though is to make a ticket to prepend the ECR url to the image under the hood and users would just do something like `image: evergreen/adl:<sha>` in their YAML

Reply:
Hmm I guess it's fine to leave it as-is then. I don't think there's any specific concerns about the account ID anyways.